nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# default controllers/views created. Home page changed to Products Inde…

Fixes:
-  N/A

Changes proposed in this request & reasons behind organizational or architectural decisions:
- N/A

Steps to test:
- Pull/Merge branch
- Start Bangazon
- Ensure Products are listed on the home page. 

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
- N/A

Commit message:
- "default controllers/views created. Home page changed to Product s Index View."

Link to feature ticket:
- N/A

@mighty-mahi-mahi
